src/main/scala/org/zalando/nakadi/client/actor/PartitionReceiver.scala

```diff
@@ -183,7 +182,7 @@ class PartitionReceiver private (val endpoint: URI,
             depth -= 1
 
             if (depth == 0 && bout.size != 0) {
-              val streamEvent = objectMapper.readValue(bout.toByteArray, classOf[SimpleStreamEvent])
+              val streamEvent = JsonParser(bout.toByteArray).convertTo[SimpleStreamEvent]
```

Comment:
Spray seems to be really nice but I heard that its maintenance has been constrained to major bugs only